src/screens/DetailDomain/elements/DetailDomainScreenHeader.js

```diff
@@ -85,7 +86,7 @@ const styles = StyleSheet.create({
   wrapperText: {
     backgroundColor: COLORS.white,
     borderRadius: 8,
-    borderColor: COLORS.anon_primary,
+    borderColor: COLORS.signed_primary,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Changing the borderColor from `COLORS.anon_primary` to `COLORS.signed_primary` could potentially affect the visual consistency across the application. Ensure this change aligns with the overall design system and color scheme of the application.



<!-- This is an auto-generated comment by OSS CodeRabbit -->